### PR TITLE
Add draft of 2021-03-29 leadership meeting minutes

### PR DESCRIPTION
**Pull request by jcoliver**
Draft of the minutes for public consumption for the 2021-03-29 UTC leadership meeting.

### CODE REVIEW

File: leadership_minutes/2021-03-29-leadership-minutes.md [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+    - [open from previous group](https://github.com/carpentries/trainers/issues/50)
+    - Trainer meeting coordination documentation
+    - Trainer Agreement
+    - IT curriculum maintainance policy
+        - Curriculum update Core team project timeline (Karen)
+        - do we want a policy about the onboarding process in place before or after this onboarding?
+    - Bonus Modules?
+    - Trainer Training?
+    - community building activities
+    - checkout and piloting
```
Comment (brownsarahm) [inline leadership_minutes/2021-03-29-leadership-minutes.md:98]:
```suggestion
6. ~5~ Projects, Content: delegate here (Sarah, 15min)
    - Sarah: per above decision I will create proposals about doing each of these ideas and we'll delegate tasks at the next meeting
```

File: leadership_minutes/2021-03-29-leadership-minutes.md [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+    - Arin: This can be a good candidate for putting Martha's rules in practice
+    - Amanda: Volunteering to create GitHub issue for this.
+5. ~4~ Projects, Procedurally: make decisions (Sarah, 10 min)
+    - multiple issues at different staged?
+    - brainstorm in meetings and then delegate?
+    - mixture?
+    - actions that are not policy/documentation? (next time)
+    - Scoping proposals within Martha's rules is an open question. For example, if we need to have a policy for maintianing the IT curriculum, do we propose to have the policy, or do we start with proposal for the policy?
+    - Mark: Don't think it makes sense to put in a full proposal if the group doesn't agree that we're going to deliberate on it. Should ask the group first if this is something we will address.
+    - Arin: +1
```
Comment (brownsarahm) [inline leadership_minutes/2021-03-29-leadership-minutes.md:87]:
```suggestion
    - Arin: +1
    - Sarah: ok, we will proceed with small proposals for project ideas that then we can move into full proposals after. 
```

**Review by brownsarahm — CHANGES_REQUESTED** [review]
I want to update the discussion on the project items

**Comment by jcoliver** [comment]
I somehow managed to remove @marklcrowe from Reviewers list when requesting re-review and now I cannot, for the life of me, figure out how to add him back to the list of Reviewers. I don't even think I can @ properly... :raised_eyebrow:

**Comment by karenword** [comment]
Neither can I... @maneesha any thoughts on why we can't request review from someone who has accepted maintainer status in this repo?

**Comment by maneesha** [comment]
@marklcrowe  was removed as an individual maintainer and added to the team.  @jcoliver  may have tried to tag him in that in-between switch over time.  Mark has been added as a reviewer to this issue.

**Review by ACharbonneau — APPROVED** [review]

**Review by brownsarahm — APPROVED** [review]

**Comment by marklcrowe** [comment]
Hi all, sorry for being a bit unresponsive about this - I've had a bit of
time off over Easter, and also must admit that I'm struggling with git. I'm
not sure what I'm supposed to do as a reviewer or what I would need to do
if I had any corrections to suggest (not that I do on this occasion). Can
someone give me some pointers please? I'm also not sure of the relationship
between the trainers-leadership section and the trainers repository - are
these minutes already public to the full trainer community? Not that's an
issue, but I just wanted to understand whether the minutes are semi-public
even in their draft format.

Also in following up the story of my being the recipient of accidental
removal and intentional reinstatement, I discovered that the Trainers
Leadership team is currently missing Arin but contains Christina K. I don't
know Christina, but should she be part of this group? And presumably we
should include Arin - what happened to him along the way?

Cheers

Mark

**Comment by brownsarahm** [comment]
@maneesha Can you confirm that Arin has been invited, but not accepted?  and can you remove Christina?

**Review by marklcrowe — APPROVED** [review]

**Review by arinbasu — APPROVED** [review]

**Pull request merged**